Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
<p>I dont use this either. Do either of you? If so can you add the bullet points</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>